libraries/rush-lib/src/utilities/Utilities.ts

```diff
@@ -664,7 +666,12 @@ export class Utilities {
       options.initialEnvironment = process.env;
     }
 
+    // Set some defaults for the environment
     const environment: IEnvironment = {};
+    if (options.pathOptions?.rushJsonFolder) {
+      environment.RUSHSTACK_FILE_ERROR_BASE_FOLDER = options.pathOptions.rushJsonFolder;
```

Comment:
If someone has this set explicitly in their own environment, it may be annoying if we override it during a Rush build.

Reply:
It gets overridden below by the existing environment variables. These are just "defaults" for Rush.